### PR TITLE
Restoring Kiali and Jaeger tutorials

### PR DESCRIPTION
**Pull request by JStickler**
This PR
- Restores the Kiali and Jaeger tutorials now that we have permission to point to the Bookinfo example application on GitHub
- Adds “Bookinfo not supported” note to Kiali and Jaeger tutorials
- Moves the tutorial files to Day_Two in the navigation.
- Corrects some namespace references from “istio-system” to “bookinfo”.
- Adds how to access the routes from the OpenShift console.
- Standardizes some text between the two tutorials.
- Addresses https://github.com/openshift/openshift-docs/issues/16598
- https://issues.jboss.org/browse/OSSM-44  (Installing Bookinfo has wrong namespace)
- https://issues.jboss.org/browse/OSSM-45 Installing Bookinfo – Note required Permissions)

@brianredbeard, @gbaufake, @mattmahoneyrh, @kevinearls, @jkandasa  Please sanity check and make sure I have the correct namespaces and routes.  Please and thank you.

### CODE REVIEW

**Comment by JStickler** [comment]
Changed references in example route output from "istio-system" to "openshift-operators"

**Comment by gbaufake** [comment]
LGTM

**Comment by brianredbeard** [comment]
Can we get the netlify site rebuilt?

**Comment by JStickler** [comment]
@brianredbeard it rebuilds every time I push an update.  The changes are there.

**Comment by brianredbeard** [comment]
In that case, this appears to be a bug:

https://github.com/openshift/openshift-docs/pull/16615/files#diff-3ca8d200ec78a8263de66058bc7493ceR40

**Comment by brianredbeard** [comment]
Also, there are a number of instances of "istio-system" on this page:

https://postga_02--ocpdocs.netlify.com/openshift-enterprise/latest/service_mesh/service_mesh_install/installing-ossm.html

**Comment by JStickler** [comment]
@brianredbeard , first comment is valid, I missed one instance in the bookinfo install.  The other link, currently the documentation has you manually create the istio-system namespace and use that for the control plane into that namespace.  So not technically a bug.  If you want to change that, the next update I was planning was to clean up the control plane section of the docs and I can do that in that PR.

File: modules/ossm-tutorial-bookinfo-install.adoc [hunk 1]
```diff
@@ -37,12 +37,18 @@ $ oc new-project bookinfo
...
+. Click the *{ProductName}* Operator.
 
-. Click *Create Service Mesh Member Roll*.
+. Click the *Istio Service Mesh Member Roll* link.
 
+.. If you have already created a Istio Service Mesh Member Roll, click the name, then click the YAML tab to open the YAML editor.
+
+.. If you have not created a Istio Service Mesh Member Roll, click *Create Service Mesh Member Roll*.
++
+NOTE: You need cluster-admin rights to edit the  Istio Service Mesh Member Roll.
```
Comment (sheriff-rh) [inline modules/ossm-tutorial-bookinfo-install.adoc]: Should this be a proper `[NOTE]` instead?

**Review by sheriff-rh — COMMENTED** [review]
One nit (more of a question), otherwise looks great!

File: modules/ossm-tutorial-bookinfo-install.adoc (hunk 1 above)
Comment (JStickler) [reply modules/ossm-tutorial-bookinfo-install.adoc]: Fixed!

**Comment by brianredbeard** [comment]
Fair enough.  LGTM then.

**Pull request merged**